content/working_with/console/widgets/secrets.md

```diff
@@ -17,7 +17,8 @@ If the secret’s value is hidden and the logged-in user isn’t the secret’s
 ![hidden-value-user]( /images/ui/widgets/hidden_secret_unauth_user.png )
```

Comment:
Probably worth to update two Secrets widget screenshots used in this page to be up to date.

Reply:
I noticed that there's separate ticket about updating Secrets documentation: https://cloudifysource.atlassian.net/browse/RD-6402 do you think I should update images as a part of this PR?

Reply:
Right, let's keep it for RD-6402.